poll/poll.py

```diff
@@ -560,26 +562,27 @@ class SurveyBlock(PollBase):
     block_name = String(default='Poll')
```

Comment:
As we decided not need to mark user changeable data with `runtime i18n`. @mattdrayer your thoughts here please ?